### ISSUE TITLE
Verify the expectations of Name checkbox-label-multiple-label and its alternative

### ISSUE THREAD

**Issue by joanmarie** [WhatSock/w3c-alternative-text-computation#12]
I have a question regarding the results for this test case:

```
if given
  <label>This <input type="checkbox" id="test" /> is</label>
  <label for="test">a test</label>
then the accessible name of the element with id of "test" is "a test"
```

[The WhatSock version of the above test case](https://whatsock.github.io/w3c-alternative-text-computation/Proposed%20Name%20and%20Description%20Tests/Name%20checkbox-label-multiple-label)

Set current node to input with ID of test (the input) and start algorithm:
* 2A doesn't apply because the input is not hidden
* 2B doesn't apply because the input lacks an aria-labelledby attribute
* 2C doesn't apply because the input lacks an aria-label attribute
* 2D does apply because the input's native markup provides an element that defines a text alternative

2D says: "return that alternative in the form of a flat string as defined by the host language, unless the element is marked as presentational" The element is not market as presentational. So what does the host language state/define? Section 5.1.1 of the [HTML AAM](https://w3c.github.io/html-aam/#input-type-text-input-type-password-input-type-search-input-type-tel-input-type-url-and-textarea-element), item 2 says:

> Otherwise use the associated label element(s) accessible name(s) - if more than one label is associated; concatenate by DOM order, delimited by spaces.

There is more than one label associated: the label containing the checkbox and the label with the for attribute. Processing them in DOM order would, I think, result in the full label being "This is a test" rather than "a test".

I think we have a similar situation with the alternative version of this test case:

```
if given
  <label for="test">a test</label>
  <label>This <input type="checkbox" id="test" /> is</label>
then the accessible name of the element with id of "test" is "a test"
```

[The WhatSock version of the alternative test case](https://whatsock.github.io/w3c-alternative-text-computation/Proposed%20Name%20and%20Description%20Tests/Name%20checkbox-label-multiple-label-alternative)

The algorithm work seems like it would be the very same, so we have to do what 2D states, namely return both labels in DOM order separated by a space. The only difference with this alternative is that "a test" preceeds "This is". So I would expect the label to be "a test This is".

**Comment by accdc** [comment]
This is fixed in AccName executable 1.18, testable using the same urls above.

**Comment by joanmarie** [comment]
Thanks! The wiki is updated. Will do the same with WPT.

Closing this issue as fixed.
